src/prefect/agent/ecs/agent.py

```diff
@@ -180,7 +180,7 @@ def __init__(  # type: ignore
         # See https://boto3.amazonaws.com/v1/documentation/api/latest/guide/retries.html
         # for more info.
         boto_config = Config(**botocore_config or {})
-        if not boto_config.retries:
+        if not boto_config.retries and not os.environ.get("AWS_RETRY_MODE"):
```

Comment:
would we want to actually set the value from the environment in `boto_config` here? or is this environment variable picked up later in the process?